modules/nf-core/bismark/align/main.nf

```diff
@@ -22,14 +22,42 @@ process BISMARK_ALIGN {
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
`bismark` has a `--prefix` flag so this should be kept.

Reply:
Never noticed it before, will add back 👍🏻